cou/apps/base.py

```diff
@@ -83,14 +83,11 @@ class OpenStackApplication:
     :type units: list[ApplicationUnit]
     :raises ApplicationError: When there are no compatible OpenStack release for the
         workload version.
-    :raises MismatchedOpenStackVersions: When units part of this application are running mismatched
-        OpenStack versions.
-    :raises HaltUpgradePlanGeneration: When the class halts the upgrade plan generation.
```

Comment:
non-blocking: why are we removing these exceptions docstrings? I'm curious about the general expectation of what and what not to include in the class docstring.

Reply:
Anything, what __init__, __new__ or __post_init__ can raised.